src/main/java/entity/PasswordValidator.java

```diff
@@ -3,4 +3,5 @@
 public interface PasswordValidator {
     public boolean passwordIsValid(String password);
 
+    String getRule();
```

Comment:
Validator service seems to work as intended (through testing a use case of less than 5 characters)!